proxy/http/HttpConfig.cc

```diff
@@ -1224,12 +1224,16 @@ HttpConfig::reconfigure()
 
   params->oride.connect_attempts_max_retries             = m_master.oride.connect_attempts_max_retries;
   params->oride.connect_attempts_max_retries_dead_server = m_master.oride.connect_attempts_max_retries_dead_server;
-  params->oride.connect_attempts_rr_retries              = m_master.oride.connect_attempts_rr_retries;
-  params->oride.connect_attempts_timeout                 = m_master.oride.connect_attempts_timeout;
-  params->oride.post_connect_attempts_timeout            = m_master.oride.post_connect_attempts_timeout;
-  params->oride.parent_connect_attempts                  = m_master.oride.parent_connect_attempts;
-  params->per_parent_connect_attempts                    = m_master.per_parent_connect_attempts;
-  params->parent_connect_timeout                         = m_master.parent_connect_timeout;
+  if (m_master.oride.connect_attempts_rr_retries >= params->oride.connect_attempts_max_retries) {
+    Warning("connect_attempts_rr_retries is greater than connect_attempts_max_retries, this means requests will never redispatch "
```

Comment:
I'll add the values-- If this condition is true it will "retry" but it won't re-do the DNS lookup (redispatch). So maybe I should word it as "it will only retry, not redo the DNS lookup"?